lib/deck.rb

```diff
@@ -0,0 +1,23 @@
+class Deck
+
+  attr_reader :cards, :count, :cards_in_category
```

Comment:
since you have a method for cards_in_category, you don't need to have this attr_reader. We only create attr_readers when we have an attribute that we want to be able to read outside of the class. Since you have handmade a method for cards_in_category, this attr_reader is moot.